tests-e2e/ocw-ci-test-course/nav-links.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+
+test("Navbar links working properly", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("")
+
+  const navbar = await page
+    .locator("#course-main-content div")
+    .filter({
+      hasText:
+        "Section 1 Menu Title Subsection 1a Menu Title Subsection 1b Menu Title Section 2",
+    })
+    .nth(3)
+  const linkCount = await navbar.locator("a").count()
+  for (let i = 0; i < linkCount; i++) {
+    const link = await navbar.locator("a").nth(i)
+    const url = await link.getAttribute("href")
+    const goto_url = url?.replace("/courses/ocw-ci-test-course/", "")
```

Comment:
Use camelCase for identifiers in TS.
```suggestion
    const gotoUrl = url?.replace("/courses/ocw-ci-test-course/", "")
```